pkg/apis/config/trusted_resources.go

```diff
@@ -29,6 +29,7 @@ import (
 // TrustedResources holds the collection of configurations that we attach to contexts.
 // Configmap named with "config-trusted-resources" where cosign pub key path and
 // KMS pub key path can be configured
+// Deprecated.
```

Comment:
I'm surprised you don't need to re-run apigen here, what's going on?

Reply:
I think this is config not api in v1beta1 or v1? when I run `update-openapigen.sh` nothing happens

Reply:
Ah I see, we only generate deepcopy functions, thanks!